src/packages/client/src/runtime/transformDmmf.ts

```diff
@@ -350,22 +372,25 @@ function getScalarArgs(
   type: string[],
   isList = false,
   kind: DMMF.FieldKind = 'scalar',
+  isNullable: boolean,
 ): DMMF.SchemaArg[] {
-  return names.map(name => getScalarArg(name, type, isList, kind))
+  return names.map((name) => getScalarArg(name, type, isList, kind, isNullable))
 }
 
 function getScalarArg(
   name: string,
   type: string[],
   isList,
   kind: DMMF.FieldKind = 'scalar',
+  isNullable: boolean,
 ): DMMF.SchemaArg {
   return {
     name,
     isRelationFilter: kind === 'object',
-    inputType: type.map(t => ({
+    inputType: type.map((t) => ({
       isList,
       isRequired: false,
```

Comment:
isRequired === false always here? Not sure I understand why 🤔

Reply:
Because this is a generated type for the `where` arguments.

Reply:
They're always optional